src/panels/config/backup/ha-config-backup-details.ts

```diff
@@ -230,87 +226,108 @@ class HaConfigBackupDetails extends LitElement {
+                                            domain,
+                                            type: "icon",
+                                            useFallback: true,
+                                            darkOptimized:
+                                              this.hass.themes?.darkMode,
+                                          })}
+                                          crossorigin="anonymous"
+                                          referrerpolicy="no-referrer"
+                                          alt=""
+                                          slot="start"
+                                        />
+                                      `
+                              }
                               <div slot="headline">${name}</div>
-                              <div slot="supporting-text">
-                                <span
-                                  class="dot ${success ? "success" : "error"}"
-                                >
-                                </span>
-                                <span>
-                                  ${success
-                                    ? this.hass.localize(
-                                        "ui.panel.config.backup.details.locations.backup_stored"
-                                      )
-                                    : this.hass.localize(
-                                        "ui.panel.config.backup.details.locations.backup_failed"
-                                      )}
-                                </span>
+                                ${
+                                  failed
+                                    ? html`
+                                        <div slot="supporting-text">
+                                          <span class="dot error"></span>
+                                          <span>
+                                            ${this.hass.localize(
+                                              "ui.panel.config.backup.details.locations.backup_failed"
+                                            )}
+                                          </span>
+                                        </div>
+                                      `
+                                    : unencrypted
+                                      ? html`
+                                          <div slot="supporting-text">
+                                            <span class="dot warning"></span>
+                                            <span> Unencrypted </span>
+                                          </div>
+                                        `
+                                      : html`
+                                          <div slot="supporting-text">
+                                            <span class="dot success"></span>
+                                            <span> Encrypted </span>
+                                          </div>
```

Comment:
div and dot is always there, I would suggest a condition for the span class and for the text